python-frontend/src/main/protobuf/symbols.proto

```diff
@@ -74,13 +74,14 @@ message ClassSymbol {
   repeated string super_classes = 4;
   repeated FunctionSymbol methods = 5;
   repeated OverloadedFunctionSymbol overloaded_methods = 6;
-  bool has_decorators = 7;
-  bool has_metaclass = 8;
-  bool is_enum = 9;
-  bool is_generic = 10;
-  bool is_protocol = 11;
-  optional string metaclass_name = 12;
-  repeated string valid_for = 13;
+  repeated VarSymbol vars = 7;
+  bool has_decorators = 8;
+  bool has_metaclass = 9;
+  bool is_enum = 10;
+  bool is_generic = 11;
+  bool is_protocol = 12;
+  optional string metaclass_name = 13;
+  repeated string valid_for = 14;
```

Comment:
I think usually in protobuf we shouldn't change field numbers to guarantee backward compatibility. I would suggest to add the new field after `valid_for` with field number `14`.
It's true that we own both serializer and deserializer, so there shouldn't be risk in changing field numbers, however I think it's a good practice not to change those.
In addition, I think that many of changes we see in protobuf binaries might be bacause of this.